application/core/Survey_Common_Action.php

```diff
@@ -850,23 +850,19 @@ function _surveybar($aData)
 
             $aData['gid'] = $gid; // = $this->input->post('gid');
 
-            if (Permission::model()->hasSurveyPermission($iSurveyID, 'surveycontent', 'read'))
-            {
+            if (Permission::model()->hasSurveyPermission($iSurveyID, 'surveycontent', 'read')) {
```

Comment:
It's not "our" convention.
It's PHP convention:
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md